packages/@ember/-internals/glimmer/lib/glimmer-tracking-docs.ts

```diff
@@ -196,43 +196,46 @@
     }
   ```
 
-  Now the sortedGuests getter will be cached based on autotracking.
+  Now the `sortedGuests` getter will be cached based on autotracking.
   It will only rerun and create a new sorted array when the guests tracked
   property is updated.
 
+
   ### Tradeoffs
 
   Overuse is discouraged.
 
   In general, you should avoid using `@cached` unless you have confirmed that
```

Comment:
_while we're here_, should we mention object reference integrity?

the current text talks about primitive values -- but how about object-values?

each time a getter is accessed and an object is returned, those are different objects.

:thinking:

Reply:
True, but I can't think about a use case that we could help by documenting a solution / explanation.
Do you have a example at hand ?
All I'm coming up with is related to 'tracking' only, with no specifics things about the 'caching' parts of it.